content/docs/deployment/csmoperator/_index.md

```diff
@@ -152,6 +153,7 @@ To perform an offline installation, the following steps should be performed:
 >NOTE: It is recommended to use the same build tool for packing and unpacking of images (either docker or podman).
 
 #### Building an offline bundle
+>NOTE: Please login to `registry.redhat.io` registry using Redhat credentials before you proceed with offline bundle creation.
```

Comment:
There is this tendency to be over polite in documenting steps. You are providing steps for a procedure not having a conversation. You do not need to start sentences with please. Just state "Login to ..." and that is more concise.

Reply:
Done